### ISSUE TITLE
Regression testing for enabledProperty

### ISSUE THREAD

**Issue by pixelzoom** [phetsims/QA#572]
In https://github.com/phetsims/sun/issues/585,  PhET UI components were converted to use the `EnabledNode` mixin, which provides a uniform API for `enabledProperty`.

The following UI components were changed, and should be regression tested:

sun
- [x] Checkbox
- [x] ComboBox
- [x] NumberSpinner
- [x] Slider

sun/buttons
- [x] push buttons
- [x] toggle buttons
- [x] momentary buttons
- [x] AquaRadioButtonGroup
- [x] RectangularButtonGroup

scenery-phet
- [x] FineCoarseSpinner
- [x] NumberControl
- [x] NumberPicker
- [x] TimeControlNode

Each component should be spot-checked in situations where the component is disabled (i.e, `enabledProperty` is exercised).  Many of these components can be tested in sun demo or scenery-phet demo. Look for an "enabled" checkbox accompanying the component's demo.

@ariel-phet has good familiarity with all PhET sims, and requested that this issue be assigning to him.

**Comment by ariel-phet** [comment]
Notes as I test (will take screen snips in the next comment of items that should be noted)

1. Time control node I think looks much better (and clearer contrast) with a "grayscale strategy" when disabled (Gas properties is a good example IMO). It looks like the step button got a bit smaller (when I compare to published gas properties). Gravity and Orbits might be the best comparison to adjust from. 

2. Something like the Eraser works really well with the opacity disabled strategy in Gas Properties (I think it is cleaner and better looking) -- that reinforces that we will want the ability to have an option either way. I think "grayscale" likely makes the most sense as a default from what I have seen. (The "Remove Divider" example in the Diffusion screen to me is a clear example of where grayscale is a better choice)

3. Something going on with the "Clear" pushbutton in GAO (no longer centered)

4. Disabled opacity for AquaRadioButtonGroup different between master and publish Gas Properties

5. The disabled checkbox with an icon (in something like Function Builder) looks cleaner in master (minor difference, but a bit less washed out). The disabled "show/hide" button in Function Builder looks a bit different, but I don't think it is an issue (a bit less of a shadow)

**Comment by ariel-phet** [comment]
@pixelzoom I only see a few regressions worth note, in order of severity:

###TimeControlNode

Wave Interference (Published)
![WI-published](https://user-images.githubusercontent.com/4051174/97770835-7e6bb200-1afc-11eb-93a8-16ce411b09c1.PNG)

Wave Interference (Master)
![WI-mater](https://user-images.githubusercontent.com/4051174/97770840-8af00a80-1afc-11eb-9722-c561d892c407.PNG)

**Comment by pixelzoom** [comment]
Re the regressions that @ariel-phet reported in https://github.com/phetsims/QA/issues/572#issuecomment-719880346...  On Slack, we discussed how to address them.

TimeControlNode differences noted for WI and GAO appear to be related to changes to TimeControlNode or the sim, not sun buttons.  TimeControlNode looks the same in scenery-phet demo before and after sun-button changes.   So these regressions will be investigated by the responsible developer (@samreid) in phetsims/wave-interference#506 and phetsims/gravity-and-orbits#365.

AquaRadioButtonGroup change in Gas Properties is "do nothing".   And to my eye, master looks a little better than published.

Layout of the "Clear" button in GAO is not related to sun button changes. It will be investigated by the responsible developer (@samreid)  in phetsims/gravity-and-orbits#364.

Eyeball button in Function Builder is "do nothing".  This is a result of 'disabled' using opacity instead of grayscale in sun buttons.  When grayscale filtering is available in scenery, its use for buttons will be addressed/standardized project wide.

That addresses all regressions. Closing.
